### DISCUSSION TITLE
whats the minium php version your package needs?

### DISCUSSION THREAD

**Discussion by michabbb** [wrk-flow/php-get-typed-value#10] [Q&A] [answered]
maybe i missed something, but i don´t see anything in the composer.json 🤔

**Comment by pionl** [comment, marked as answer]
Good catch. Its missing in composer.json.

I've added it only as badge...

I's PHP 8.1.

In near future I would like to experiment with https://getrector.org/blog/how-all-frameworks-can-bump-to-php-81-and-you-can-use-older-php

**Reply by michabbb** [reply]
nice article 😏 sadly my legacy 💩 code still is on PHP 7.4 🙈 so I guess my testing will take a bit more time than expected 😢

**Reply by pionl** [reply]
I know the feeling, but I was able to upgrade large codebase from 7.2 to PHP 8.1. My code was not the tricky part, but we had to update some dependencies.

PHPStan fill help you catch some bugs (even lower level). :)

**Reply by michabbb** [reply]
it´s always the dependencies 🙈 I am already using psalm, also a huge help 😏
